pkg/agent/controller/networkpolicy/cache.go

```diff
@@ -67,7 +67,7 @@ type rule struct {
 	// Priority of the NetworkPolicy to which this rule belong. nil for K8s NetworkPolicy.
 	PolicyPriority *float64
 	// Priority of the tier that the NetworkPolicy belongs to. nil for K8s NetworkPolicy.
-	TierPriority *v1beta1.TierPriority
+	TierPriority *uint32
```

Comment:
I think typically K8s APIs will use `int32` and not `uint32`. I think we should keep it consistent with the rule `Priority` above.

I believe that the rationale for using `int32` in general is:
* some programming languages don't have native support for unsigned integers
* it's easier to catch sign errors with signed integers, depending on the programming language (it it's negative, it's clearly an error but if it's a very large number, it could be normal or it could be a negative number overflow)